packages/order-watcher/src/order_watcher/order_watcher.ts

```diff
@@ -241,7 +242,15 @@ export class OrderWatcher {
             return;
         }
         const log = logIfExists as LogEntryEvent; // At this moment we are sure that no error occured and log is defined.
-        const maybeDecodedLog = this._web3Wrapper.abiDecoder.tryToDecodeLogOrNoop<ContractEventArgs>(log);
+        let maybeDecodedLog;
+        try {
+            maybeDecodedLog = this._web3Wrapper.abiDecoder.tryToDecodeLogOrNoop<ContractEventArgs>(log);
+        } catch (error) {
+          if (error.code === ethers.errors.INVALID_ARGUMENT) {
```

Comment:
Can you add a `// HACK: Some description` comment above this explaining the rationale for ignoring these errors for the next engineer who chances upon this?